tests/factories.py

```diff
@@ -18,10 +18,10 @@ class Meta:  # pylint: disable=too-few-public-methods
 
     id = factory.Sequence(lambda n: n)
     name = factory.Faker("first_name")
-    user_id = FuzzyChoice(choices=["1", "2", "3", "4"])
-    product_id = FuzzyChoice(choices=["1", "2", "3", "4"])
-    quantity = FuzzyChoice(choices=[1.0, 2.0, 3.0, 4.0])
-    price = FuzzyChoice(choices=[1.0, 2.0, 3.0, 4.0])
+    user_id = str(FuzzyInteger(1, 999))
```

Comment:
Could you explain why here we should use FuzzyInteger instead of FuzzyChoice? I don't really get it.

Reply:
I think we can have more test cases.

Reply:
Sure, makes sense.

Reply:
May I roll-back the codes until we have a better method?
Testing only four numbers doesn't seem like a good test.

Reply:
I agree, this part is good to go. For quantity, try use str(float(FuzzyInteger(1, 999)+0.5)).